config/env/default.js

```diff
@@ -26,6 +26,10 @@ module.exports = {
   domain: process.env.DOMAIN || 'localhost:3000',
   supportEmail: 'support@trustroots.org', // TO-address for support requests
   profileMinimumLength: 140, // Require User.profile.description to be >=140 chars to send messages
+  // A message below longMessageMinimumLength will be 'short' in influxdb, otherwise 'long'
+  // TODO choose an expressive value
+  // Used in ./modules/messages/server/services/message-to-influx.server.service
+  longMessageMinimumLength: 150,
```

Comment:
@simison i moved it to limits as you suggested.

I changed value to 60. (That is supposedly a median length of a tweet.) Still totally random, but 150 felt a bit long for a short message. You're free to choose whatever you want. No ego involved in this value. 😸

As long as the variable stays consistent across different settings & environment, i'm indifferent to it's place. So i can't really comment.